test_scripts/Polices/External_UCS/External_Consent_OFF/024_ATF_Policies_External_Consent_OFF_many_entities_pairs_in_one_group_01.lua

```diff
@@ -0,0 +1,223 @@
+--------------------------------------Requirement summary---------------------------------------------
```

Comment:
@maria-mih common_steps and common_functions should not be present in the script because they are described in all_common_modules